spec/features/run_complete_sequence_spec.rb

```diff
@@ -1,166 +1,169 @@
-      navigate_and_verify :page, param_assertions
-    end
-
-  end
-
-  feature "'mode' query parameter handling" do
-
-    feature "not as a teacher..." do
-      scenario "We should NOT see 'teacher-edition' in the url" do
-        verify_param_passed_through_sequence(absent: {mode:'teacher-edition'})
-      end
-    end
-
-    feature "as a teacher " do
-      feature "with `mode` set to `teacher-edition` " do
-        let(:params) { {mode: 'teacher-edition' } }
-        scenario "We should see 'mode=teacher-edition' in the url" do
-          verify_param_passed_through_sequence(present: {mode:'teacher-edition'})
-        end
-
-        scenario "We should see the Teacher View banner on the first page" do
-          visit sequence_url
-          expect(page.body).to match(teacher_edition_regex)
-        end
-
-        feature "When we are redirected to page 2 of activity 1 by our run" do
-          let(:activity) { sequence.activities.first }
-          let(:run_page) { activity.pages[1] }
-          let(:activity_url) do
-            # set the last active page
-            run1.page = run_page
-            run1.save!
-            sequence_activity_path(sequence, activity, params: params)
-          end
-
-          scenario "The redirected page retain our mode param" do
-            visit activity_url
-            # We assume that we have been redirected to
-            # /sequence/:id/activity/:id/page/:id
-            expected_url = "/sequences/#{sequence.id}/activities/#{activity.id}/pages/#{run_page.id}"
-            expect(current_url).to match(expected_url)
-            teacher_mode = "mode=teacher-edition"
-            expect(current_url).to match(teacher_mode)
-            expect(page.body).to match(teacher_edition_regex)
-          end
-
-          scenario "We click on the Sequence title and retain our mode param" do
-            visit activity_url
-            selector = "#container > header > div > div.site-logo.logo-l > div > h2 > a"
-            find(:css, selector, match: :first).click
-            teacher_mode = "mode=teacher-edition"
-            expect(current_url).to match(teacher_mode)
-            expect(page.body).to match(teacher_edition_regex)
-          end
-        end
-
-        scenario "We should see the Teacher View banner on the last page" do
-          verify_param_passed_through_sequence(present: {mode:'teacher-edition'})
-          expect(page.body).to match(teacher_edition_regex)
-          expect(current_url).to match "mode=teacher-edition"
-        end
-      end
-
-      feature "Combined with non-pass-through param `foo=bar` " do
-        let(:params) { {mode: 'teacher-edition', foo: 'bar'} }
-        scenario "We should see 'mode=teacher-edition' in the url, and not foo=bar" do
-          verify_param_passed_through_sequence(
-            present: { mode:'teacher-edition' },
-            absent:  { foo:'bar' }
-          )
-        end
-      end
-
-    end
-  end
-end
+# require 'spec_helper'
```

Comment:
@lublagg could you add a comment as to why this spec is disabled?  It will help future folks when they load this file.  Something like: 

`# NOTE: this spec is disabled because runs are no longer created when viewing activities, instead the user is redirected to the Activity Player`

Reply:
Will do!